scripts/init.sh

```diff
@@ -77,24 +77,26 @@ if [ -r $DEFAULT ]; then
     source $DEFAULT
 fi
 
-function pidofproc() {
-    if [ $# -ne 3 ]; then
-        echo "Expected three arguments, e.g. $0 -p pidfile daemon-name"
-    fi
+if ! typeset -F pidofproc &>/dev/null; then
+  function pidofproc() {
+      if [ $# -ne 3 ]; then
+          echo "Expected three arguments, e.g. $0 -p pidfile daemon-name"
+      fi
 
-    PID=`pgrep -f $3`
-    local PIDFILE=`cat $2`
+      pid=`pgrep -f '\'$3'\>'`
+      local pidfile=`cat $2`
 
-    if [ "x$PIDFILE" == "x" ]; then
-        return 1
-    fi
+      if [ "x$pidfile" == "x" ]; then
```

Comment:
These should just be spaces as I'm fairly certain my vim setup doesn't use/strips tabs, but I'll definitely double check.